numpy/core/numeric.py

```diff
@@ -1935,6 +1935,9 @@ def warn_if_insufficient(width, binwidth):
                 "will raise an error in the future.", DeprecationWarning,
                 stacklevel=3)
 
+    # Ensure that num is a Python integer.
```

Comment:
Perhaps worth adding "to prevent overflow"

Reply:
OK, done.